### ISSUE TITLE
[Clip Confirm] Easier way to delete clips

### ISSUE THREAD

**Issue by x9136** [FrankerFaceZ/Add-Ons#160]
Hello!

Creating a clip automatically saves it, but it's often useful to check something through the clip feature. As outcome, your clip will be saved in the Clips. It's not obvious. You can see a lot of random and obscure clips at any channel.

To delete a clip you have to do 8 steps: click on Userpic in the top right corner → Video Producer → Clips → Click on a clip or check it → Delete → Confirm Delete → Close Success pop-up → Leave.

It will be useful to have an option to delete a clip when you leave https://clips.twitch.tv/create. For example, I create clips only if I need to check something. Not for highlights. If your usage of the clip feature is mixed, then it's useful to have a delete button on https://clips.twitch.tv/create. Click Delete to delete and close the tab → Done. Though I don't know what is possible.

It will be nice if Twitch improve that, but seems that is not important improvement.
https://twitch.uservoice.com/forums/923368-video-features/suggestions/39801619-clip-features-cancel-save-for-later

What do you think?

**Comment by argowizbang** [comment]
I've actually looked into this in the past for this exact reason but never really found any feasible way to accomplish this, unfortunately. Granted I only did some cursory searching around so it's possible there is some kind of method to do so that I overlooked, but for now I'm not sure this is easily doable. That said, if anyone knows something I don't then I'd definitely be open to making such a feature available somehow.
